tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Wcf/WcfCommon.cs

```diff
@@ -123,15 +125,27 @@ static IEnumerable<string> GetHeaderValues(IMessageHeaders headers, string name)
 
                 string operationName = tracer.CurrentTraceSettings.Schema.Server.GetOperationNameForComponent("wcf");
                 var tags = new WcfTags();
+
+                string? resourceName = null;
+                if (useWebHttpResourceNames
+                 && requestProperties.TryGetValue("UriMatched", out object uriMatched)
+                 && uriMatched is true
+                 && requestProperties.TryGetValue("UriTemplateMatchResults", out var matchResults)
+                 && matchResults is not null
+                 && matchResults.DuckCast<UriTemplateMatchStruct>().Template is { } template)
+                {
+                    resourceName = template.ToString();
```

Comment:
In the PR description you mentioned you were going to add the HTTP verb in front of the template. It looks like that is missing, unless you have since decided against it and removed it

Reply:
_Sigh_ I failed to push my latest changes 🤦‍♂️